web/src/javascript/drawMap.js

```diff
@@ -0,0 +1,70 @@
+    var screenCanvas = $("#screenCanvas")[0];
+    var screenContext = screenCanvas.getContext("2d");
+    var screenWidth = screenCanvas.width;
+    var screenHeight = screenCanvas.height; 
+    //Static position for tests for the screen window on the mini-map
+    var xScreenPosOnMap = 200;
+    var yScreenPosOnMap = 200;    
+
+    var mapCanvas = document.createElement('canvas');
+    var mapContext = mapCanvas.getContext('2d');
+    var mapWidth = screenWidth*3;
+    var mapHeight = screenHeight*3;
+
+    var miniMapCanvas = document.createElement('canvas');
+    var miniMapContext = miniMapCanvas.getContext('2d');
+    var miniMapWidth = screenWidth/4;
+    var miniMapHeight = screenHeight/4;
+    var miniMapPosX = screenWidth-miniMapWidth;
+
+    var miniMapScreenPosWidth = miniMapWidth/3;
+    var miniMapScreenPosHeight = miniMapHeight/3;
+
+    function drawCellsOnMap(points){
+        //set dimensions
+        mapCanvas.width = mapWidth;
+        mapCanvas.height = mapHeight;
+        
+        var i, point, len = points.length;
+        for(i = 0; i < len; i += 1) {
+            point = points[i];
+            
+            mapContext.beginPath();
+            mapContext.arc(point.x, point.y, point.radius, 0, Math.PI * 2, false);
+            mapContext.fillStyle = "#ed1515";
+            mapContext.fill();
+        }
+    }
+
+
+    function drawMap(){
```

Comment:
How about `showMap()`? :) Then, `draw` would mean "draw on the canvas" and `show` would mean "display the canvas to the user".